vnext/Shared/Threading/BatchingQueueThread.h

```diff
@@ -27,12 +27,14 @@ struct BatchingQueueThread final : facebook::react::BatchingMessageQueueThread {
  private:
   void EnsureQueue() noexcept;
   void ThreadCheck() noexcept;
+  void postBatch() noexcept;
```

Comment:
>postBatch [](start = 7, length = 9)

(nit) why some private Apis follow Camel casing, while others Pascal? Would be nice to align all private Apis :)